### PR TITLE
docs: updating as per branding guidelines

### PR DESCRIPTION
**Pull request by SamKirsch10**
## Description

updating as per branding guidelines

## Issue or Ticket

n/a

## Checklist before submitting PR for review
- [ ] I have tested these changes with the included tfvars
- [ ] This change requires a doc update, and I've included it
- [ ] My code follows the style guidelines of this project
- [ ] I have ensured my code is commented and any new terraform variables have proper descriptions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +38,7 @@ script configuration can be found in [this KB article](https://help.fullstory.co
 
 ### With Route 53 Record Creation
 
-This module will automatically create the DNS records if a value for `route53_zone_name` is provided in reference to an existing Route 53 zone within the same AWS account.
+This module will automatically create the DNS records if a value for `route53_zone_name` is provided in reference to an existing Route 53 zone within hte same AWS account.
```
Comment (Oldsouldier) [inline README.md:41]: ?

Comment (SamKirsch10) [reply README.md]: 🤷

File: README.md [hunk 2]
```diff
@@ -19,13 +20,13 @@ script configuration can be found in [this KB article](https://help.fullstory.co
...
+| <a name="input_cloudfront_origin_request_policy_name"></a> [cloudfront\_origin\_request\_policy\_name](#input\_cloudfront\_origin\_request\_policy\_name) | (optional) A name to uniquely identify the cloudfront origin request policy for the relay. This is required to deploy multiple relay modules to the same AWS account, as such policies must be uniquely named. Example: `fullstory-relay-origin-request-policy-production`. | `string` | `"fullstory-relay-origin-request-policy"` | no |
+| <a name="input_relay_fqdn"></a> [relay\_fqdn](#input\_relay\_fqdn) | The fully qualified domain name for the relay. Example: `fsrelay.your-company.com`. | `string` | n/a | yes |
+| <a name="input_route53_zone_name"></a> [route53\_zone\_name](#input\_route53\_zone\_name) | (optional) The Route 53 zone name for placing the DNS CNAME record. If omitted, a value for `acm_certificate_arn` must be provided. Defaults to null. | `string` | `null` | no |
+| <a name="input_target_fqdn"></a> [target\_fqdn](#input\_target\_fqdn) | (optional) The fully qualified domain name that the relay targets. Defaults to `fullstory.com`. | `string` | `"fullstory.com"` | no |
```
Comment (Oldsouldier) [inline README.md:29]: Guessing this is a generated diff from the terraform-docs upgrade?

Comment (SamKirsch10) [reply README.md:29]: yea

**Review by Oldsouldier — APPROVED** [review]

**Pull request merged**